katsdpcontroller/product_config.py

```diff
@@ -1153,6 +1153,10 @@ def n_chans(self) -> int:
     def flags(self) -> FlagsStream:
         return cast(FlagsStream, self.src_streams[0])
 
+    @property
```

Comment:
ImageStream already has this property, why do we need to define it again ?

Reply:
Thanks, well spotted! It looks like I'm also uselessly redefining the flags property. I'll get rid of both.